v22.1/restore.md

```diff
@@ -108,6 +108,11 @@ You can exclude a table's row data from a backup using the [`exclude_data_from_b
 - All [tables](create-table.html) (which automatically includes their [indexes](indexes.html))
 - All [views](views.html)
 
+{% include_cached new-in.html version="v22.1" %} Also, a full cluster restore will:
+
+- Restore [temporary tables](temporary-tables.html) to their original database during a full cluster restore.
+- Drop `defaultdb` and `postgres` [pre-loaded databases](show-databases.html#preloaded-databases) before the restore begins. You can only restore `defaultdb` and `postgres` if they are present in the original [backup](take-full-and-incremental-backups.html).
```

Comment:
supernit: "Drop the `defaultdb` and ..."

Question: Are we dropping our live `defaultdb` and `postgres` DBs or the backup's?